### ISSUE TITLE
[Social Plugin] Concurrency `1` throws error

### ISSUE THREAD

**Issue by bilal-fazlani** [squidfunk/mkdocs-material#5515]
### Context

In the version [mkdocs-material-9.1.12-insiders-4.34.0](https://github.com/squidfunk/mkdocs-material-insiders/releases/tag/9.1.12-insiders-4.34.0) when using social plugin (with customisations) I am getting below error in netlify

```
11:01:33 AM: $ mkdocs build
11:01:33 AM: Traceback (most recent call last):
11:01:33 AM:   File "/opt/buildhome/python3.8/bin/mkdocs", line 8, in <module>
11:01:33 AM:     sys.exit(cli())
11:01:33 AM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
11:01:33 AM:     return self.main(*args, **kwargs)
11:01:33 AM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/click/core.py", line 1055, in main
11:01:33 AM:     rv = self.invoke(ctx)
11:01:33 AM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
11:01:33 AM:     return _process_result(sub_ctx.command.invoke(sub_ctx))
11:01:33 AM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
11:01:33 AM:     return ctx.invoke(self.callback, **ctx.params)
11:01:34 AM: Failed during stage 'building site': Build script returned non-zero exit code: 2 (https://ntl.fyi/exit-code-2)
11:01:33 AM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/click/core.py", line 760, in invoke
11:01:33 AM:     return __callback(*args, **kwargs)
11:01:33 AM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/mkdocs/__main__.py", line 248, in build_command
11:01:33 AM:     cfg['plugins'].run_event('startup', command='build', dirty=not clean)
11:01:33 AM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/mkdocs/plugins.py", line 522, in run_event
11:01:33 AM:     result = method(**kwargs)
11:01:33 AM:   File "/opt/buildhome/python3.8/lib/python3.8/site-packages/material/plugins/social/plugin.py", line 76, in on_startup
11:01:33 AM:     self.card_pool = ThreadPoolExecutor(self.config.concurrency - 1)
11:01:33 AM:   File "/usr/lib/python3.8/concurrent/futures/thread.py", line 141, in __init__
11:01:33 AM:     raise ValueError("max_workers must be greater than 0")
11:01:33 AM: ValueError: max_workers must be greater than 0
```

### Bug description

It was working fine in [mkdocs-material-9.1.12-insiders-4.33.2](https://github.com/squidfunk/mkdocs-material-insiders/releases/tag/9.1.12-insiders-4.33.2)

### Related links

- https://squidfunk.github.io/mkdocs-material/setup/setting-up-social-cards/?h=social#configuration

### Reproduction

I can reproduce this on local by setting `concurrency` to 1
If I set concurrency to 2, the issues goes away.

I don't have a minimal reproduction ATM, but if that is absolutely necessary I can spend time on it and get back in a day or two.

### Steps to reproduce

```yml
  - social:
      cards_layout_dir: social-preview
      cards_layout: custom
      concurrency: 1
```

```
mkdocs build
```

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting! This was actually a left-over from testing. Fixed in [`1f0db88ae`](https://github.com/squidfunk/mkdocs-material-insiders/commit/1f0db88ae2421abcf3bce380451bcb6e5847164e).

Now, the default is available CPUs - 1, with a minimum of 1.

**Comment by squidfunk** [comment]
Released as part of [9.1.13+insiders-4.34.1](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#4.34.1).
